metabase/metabase
# Make Radio accessible

Epic: https://github.com/metabase/metabase/issues/19696

- Makes Radio accessible
- Convers to TypeScript
- Adds a story

How to test:
- Open storybook `yarn storybook`
- Navigate to `Radio`
- Make sure it receives focus on tab
- Make sure you can navigate between options by pressing arrow keys
